modules/openapi-generator/src/main/resources/Java/libraries/retrofit2/ApiClient.mustache

```diff
@@ -357,7 +365,9 @@ public class ApiClient {
       throw new RuntimeException("auth name \"" + authName + "\" already in api authorizations");
     }
     apiAuthorizations.put(authName, authorization);
-    okBuilder.addInterceptor(authorization);
+    if(okBuilder != null){
```

Comment:
May be better to throw an Exception with an explicit message if okBuilder is null ?